kong/clustering/rpc/manager.lua

```diff
@@ -634,6 +634,13 @@ function _M:connect(premature, node_id, host, path, cert, key)
   end
 
   ::err::
+  local worker_events = assert(kong.worker_events)
+
+  -- notify this worker
+  local ok, err = worker_events.post_local("clustering:jsonrpc", "connection_lost")
```

Comment:
This may cause duplicated event when connection disconnected, what's your opinion? how do we distinguish them?

Reply:
I see, it is dp side event.